HelpSource/Guides/WritingClasses.schelp

```diff
@@ -334,46 +494,56 @@ method can be used instead of code::storeOn::.
 code::
 // either
 MyTestPoint {
-    var <x, <y;
+	var <x, <y;
 
-    *new{ |x,y| ^super.newCopyArgs(x,y) }
+	*new { |x, y|
+		^super.newCopyArgs(x,y)
+	}
 
-    storeOn { arg stream;
-        stream << "MyTestPoint.new(" << x << ", " << y << ")";
-    }
+	storeOn { | stream |
+		// note that <<< stands for storeOn, and << for printOn.
+		// we want x and y to be completely represented
+		stream << "MyTestPoint.new(" <<< x << ", " <<< y << ")"
+	}
 }
 
 // or
 MyTestPoint {
-    var <x, <y;
+	var <x, <y;
 
-    *new{ |x,y| ^super.newCopyArgs(x,y) }
+	*new { |x, y|
+		^super.newCopyArgs(x,y)
+	}
 
-    storeArgs { arg stream;
-        ^[x,y]
-    }
+	storeArgs { | stream |
+		^[x, y]
+	}
 }
 ::
 
 code::
-MyTestPoint(2,3).asCompileString;
+MyTestPoint(2, 3).asCompileString;
 ::
 
-section:: Catching methods that are not explicitly defined
+section::Private Methods
 
-It is possible to catch calls to methods that are not explicitly defined in a Class by overriding code::doesNotUnderstand::.
+Private methods are marked by a prefix code::pr::, e.g. code::prBundleSize::. This is just a strong::naming convention::, the message can still be called from anywhere. It is not recommended to do this.
```

Comment:
the comma here should be a semicolon ("a naming convention; the message..."). "It is not recommended to do this" is vague because it's unclear what "this" refers to. Maybe clearer as "It is recommended to stick to convention and only call private methods from within the class that defines them."

Reply:
good!